src/java.base/share/classes/java/io/File.java

```diff
@@ -1813,18 +1813,20 @@ public boolean canExecute() {
      * <p> A particular Java platform may support zero or more
      * hierarchically-organized file systems.  Each file system has a
      * {@code root} directory from which all other files in that file system
-     * can be reached.  Windows platforms, for example, have a root directory
-     * for each active drive; UNIX platforms have a single root directory,
-     * namely {@code "/"}.  The set of available filesystem roots is affected
-     * by various system-level operations such as the insertion or ejection of
-     * removable media and the disconnecting or unmounting of physical or
-     * virtual disk drives.
+     * can be reached.
      *
      * <p> This method returns an array of {@code File} objects that denote the
-     * root directories of the available filesystem roots.  It is guaranteed
+     * root directories of the existing filesystem roots.  It is guaranteed
```

Comment:
I don't think you can change "available" to "existing" here without a lot more discussion. In particular, the javadoc can't set the expectation that "existing" implies that File::exists will return true, it might not. So I think drop this from the change.

Reply:
Thanks for your comment. I removed above change.